examples/workflows/main.go

```diff
@@ -114,40 +114,23 @@ func run(ctx context.Context, cfg *config) error {
 				req, ok := event.Resource.(*types.AccessRequestV3)
 				if !ok {
 					return trace.BadParameter("unexpected resource type %T", event.Resource)
-				}
-
+				}	
 				// Gather AccessRequestUpdate params
-				params := types.AccessRequestUpdate{
-					RequestID: req.GetName(),
-					Annotations: map[string][]string{
-						"strategy": {"allow_list"},
-					},
-				}
-
-				// Searching through allowList for a user match
-				// and update the request state accordingly.
-				allowed := false
-				for _, user := range cfg.AllowList {
-					if req.GetUser() == user {
-						allowed = true
-						break
-					}
-				}
-				if allowed {
-					log.Printf("User %q in Allow List, approving request...", req.GetUser())
-					params.State = types.RequestState_APPROVED
-					params.Reason = "user in Allow List"
-				} else {
-					log.Printf("User %q not in Allow List, denying request...", req.GetUser())
-					params.State = types.RequestState_DENIED
-					params.Reason = "user not in Allow List"
-				}
-
+				params := types.AccessReviewSubmission{
+				    Resource: req.GetName(),
+					State:    types.RequestState_APPROVED,
```

Comment:
This code is not properly formatted.

https://go.dev/blog/gofmt

Reply:
Formatted the code.